src/test/java/Study/SetTest.java

```diff
@@ -0,0 +1,58 @@
+package Study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("TestSet size")
+    void testSetSize(){
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

assertj에서는 다양한 테스트 함수를 지원하고 있습니다. 상황에 맞는 함수를 찾아서 적용해보시는 것도 좋을 것 같아요 :)

Reply:
안녕하세요 
리뷰를 받은 곽성훈입니다. 꼼꼼한 리뷰 감사합니다.

리뷰 후 여러가지 피드백을 주셨는데 이 후 어떻게 진행해야 할지 몰라서 이렇게 댓글로 연락드립니다.

현재 제  Step1 브랜치의 상태는  Merge된 상태입니다.
준용님이 주신 리뷰들을 반영하여 다시 피드백을 받고 싶은 경우는 어떻게 해야하나요?

Local의 Step1 코드 수정 -> 새로운 PR생성 -> 리뷰요청 이 맞나요?

동영상 강의에는 피드백 후 Merge 안 한후 수정 후 머지인데 
이미 머지된 상태에서는 어떻게 진행해야 하는지 모르겠어서 이렇게 메일 전송드립니다.

답변 기다리고 있겠습니다.
감사합니다.